public/api-documentation/swagger.json

```diff
@@ -401,6 +441,32 @@
         }
       }
     },
+    "SubmitSignature": {
+      "type": "object",
+      "properties": {
+        "answer": {
+          "type": "string"
+        },
+        "question_id": {
+          "type": "string"
+        },
+        "token": {
+          "type": "string"
+        },
+        "user_id": {
+          "description": "CDX user id",
+          "type": "string"
+        }
+      }
+    },
+    "SubmitTransaction": {
```

Comment:
I suggested `Submission` because it was a word already in use elsewhere in the app to designate that particular milestone in the overall workflow.

Our app has never signed anything. It's always acted as a proxy for the CDX service, which does the actual signing. That doesn't mean we should not use the word "sign" or "signature" -- perfectly good words to describe the end result for the user.

If we are effectively replacing the existing signature API with this one, why not replace it instead of building a 2nd API endpoint alongside it? Otherwise we end up with 2 APIs for getting a Manifest signed. The User doesn't care about the historical reasons for this particular phase of the app (API) development.

Let's change the API to make "signature" use the new CDX pattern.